<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #eeeeee;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="show_dialog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border: 5px solid #222222;
background-color: #dede00;
font: 8pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Show dialog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Старт/стоп синхронизации</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid #222222;
border-radius:0px;
background-color: rgba(0,0,0,0);
font: 8pt &quot;Consolas&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/toggle-on.png</normaloff>icons/toggle-on.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- SetIcon example</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icons/lego.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Organo (colored version at: log&quot;;
padding:5 px;
</string>
    </property>
    <property name="text">
     <string>Welcome to PyLego demo!</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="percent">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="select_dir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select dir...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_file">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select *.py file...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 0, 0);
color: rgb(255, 255, 255);
font: 8pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>581</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="mytable">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>231</width>
       <height>200</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Organo (colored version at: log&quot;;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>16</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>17</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>18</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>20</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>21</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>22</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>23</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>24</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>25</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>26</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>27</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>28</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>29</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>30</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>31</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>32</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>33</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>34</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>35</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>36</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>37</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>38</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>39</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>41</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>first</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>second</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>third</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>74060</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>74070</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>74070</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>74060</string>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string>74050</string>
      </property>
     </item>
     <item row="4" column="2">
      <property name="text">
       <string>154</string>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="5" column="2">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QTextBrowser" name="outbox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>331</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid #999999;
background-color: #afdef8;
font: 8pt &quot;Consolas&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
 </resources>
 <connections/>
</ui>
